Tests/MultipartFormDataTests.swift

```diff
@@ -22,6 +22,7 @@
 //  THE SOFTWARE.
 //
 
+@testable
 import Alamofire
```

Comment:
Removed the `@testable`, as I moved the boolean extensions into the test target.